### ISSUE TITLE
Plugin does not load

### ISSUE THREAD

**Issue by orena1** [briangann/grafana-datatable-panel#97]
Hi, it seems that I can not activate the plugin, I've installed it as instructed:
```
 ./bin/grafana-cli --pluginsDir ".../Local/grafana-6.7.3/data/plugins/" plugins install briangann-datatable-panel
```

But I get this error when I try to use it:
```
Plugin component error TypeError: Cannot read property 'dataReceived' of undefined
    at e [as constructor] (ctrl.ts:131)
    at Object.invoke (angular.js:5040)
    at M.instance (angular.js:11000)
    at it (angular.js:9852)
    at angular.js:9174
    at angular.js:9039
    at variableQueryEditorLoader.tsx:23
    at d (ChangeTracker.ts:14)
    at ChangeTracker.ts:25

```

See here:
![image](https://user-images.githubusercontent.com/8983713/81842224-54c1cd00-9554-11ea-9423-73ee2bb907ab.png)

Any help?
Thanks.

**Comment by orena1** [comment]
Same as this - 
https://github.com/grafana/grafana-polystat-panel/issues/78

**Comment by briangann** [comment]
taking a look with 6.7.3, that build should have support for the event names.

**Comment by orena1** [comment]
I've upgraded from a lower version using these instructions:
[](https://grafana.com/docs/grafana/latest/installation/upgrading/
)> Upgrade from binary .tar file
> If you downloaded the binary .tar.gz package, then you can just download and extract the new package and overwrite all your existing files. However, this might overwrite your config changes.
> 
> We recommend that you save your custom config changes in a file named <grafana_install_dir>/conf/custom.ini. This allows you to upgrade Grafana without risking losing your configuration changes.

**Comment by briangann** [comment]
i'll add additional checks for backwards compatiblity, converted to the new event types too soon!

**Comment by briangann** [comment]
fixed in 1.0.2

**Comment by orena1** [comment]
Thanks
